<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetsImportView</class>
 <widget class="QWidget" name="WidgetsImportView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="WidgetsComboBoxLayout" name="cbLocale">
       <property name="title">
        <string>Locale</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="WidgetsComboBoxLayout" name="cbGroup">
       <property name="title">
        <string>Grouping symbol</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="WidgetsComboBoxLayout" name="cbDecimal">
       <property name="title">
        <string>Decimal separator</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="WidgetsLineEditLayout" name="leDateTime">
       <property name="title">
        <string>Date Time Format</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="WidgetsLineEditLayout" name="lbDatePreview">
       <property name="title">
        <string>Example</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="BtnFlip">
       <property name="text">
        <string>Transpose</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ShowPreview">
       <property name="text">
        <string>Preview</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="WidgetsTableViewBase" name="SourceTable">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
     </widget>
     <widget class="WidgetsTableViewBase" name="PreviewTable">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetsLineEditLayout</class>
   <extends>QFrame</extends>
   <header>WidgetsModule/internal.hpp</header>
  </customwidget>
  <customwidget>
   <class>WidgetsComboBoxLayout</class>
   <extends>QFrame</extends>
   <header>WidgetsModule/internal.hpp</header>
  </customwidget>
  <customwidget>
   <class>WidgetsTableViewBase</class>
   <extends>QTableView</extends>
   <header>WidgetsModule/internal.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
